stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptions.kt

```diff
@@ -202,7 +205,8 @@ public fun defaultMessageOptionsState(
         } else {
             null
         },
-        if (((isOwnMessage && canEditOwnMessage) || canEditAnyMessage) && !selectedMessage.isGiphy()) {
+        if (visibility.isEditMessageVisible &&
```

Comment:
`klint` says that these conditions are too complex. I see we have similar conditions in other places. Should I try to simplify these ones or we can ignore them?

Reply:
We should simplify AFAP.
In the case it can't be simplified (Well, it can always be simplified, but sometimes it requires some refactoring that doesn't make sense on the actual PR), we can suppress the warning or update detekt baseline files to avoid this warning, because check style check is mandatory before merging the PR